internal/appv2/patches.go

```diff
@@ -198,6 +198,23 @@ func _patchService(service map[string]any) (map[string]any, error) {
 		}
 	}
 
+	if rawInternalPort, ok := service["internal_port"]; ok {
+		switch cast := rawInternalPort.(type) {
+		case string:
+			n, err := strconv.Atoi(cast)
+			if err != nil {
+				return nil, fmt.Errorf("Can not convert internal_port '%s' to integer: %w", cast, err)
+			}
+			service["internal_port"] = n
+		case float64:
+			service["internal_port"] = int(cast)
+		case int64:
```

Comment:
I think int16, int32, int, and the uint variations would also all be acceptable to cast to int.

Reply:
You're right, got them all